### PR TITLE
Fix Optimism signatures spell

### PR DESCRIPTION
**Pull request by soispoke**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/addresses/ethereum/addresses_ethereum_dex.sql [hunk 1]
```diff
@@ -41,6 +41,7 @@ FROM (VALUES
       ('0x7a250d5630b4cf539739df2c5dacb4c659f2488d', 'Uniswap', 'Router02'),
       ('0xe592427a0aece92de3edee1f18e0157c05861564', 'Uniswap', 'V3 Router'),
       ('0x68b3465833fb72a70ecdf485e0e4c7bd8665fc45', 'Uniswap', 'V3 Router 2'),
+      ('0xb45a2dda996c32e93b8c47098e90ed0e7ab18e39', 'Uniswap', 'TransitSwapRouterV4'),
```
Comment (0xRobin) [inline models/addresses/ethereum/addresses_ethereum_dex.sql:44]: hmm @soispoke why does it include this change?

Comment (soispoke) [reply models/addresses/ethereum/addresses_ethereum_dex.sql:44]: no idea!

Comment (soispoke) [reply models/addresses/ethereum/addresses_ethereum_dex.sql]: should be good now

**Comment by 0xRobin** [comment]
CI complained about the columns.
In ethereum.signatures we only have these:
[id, signature, abi, type, created_at]

**Review by 0xRobin — APPROVED** [review]

**Pull request merged**